release-1.8/release_notes_draft.md

```diff
@@ -268,17 +268,18 @@ This section provides an overview of deprecated API versions, options, flags, an
 
 ## **Notable Features**
 
-### [Workload API (apps/v1beta2)](https://github.com/kubernetes/features/issues/353)
+### Workload API (apps/v1beta2)(https://github.com/kubernetes/features/issues/353)
```

Comment:
Instead of giving hyperlinks to the title (none of the headings have hyperlinks in the RN), we could add a sentence ` For more information on Workload API, see (apps/v1beta2)(https://github.com/kubernetes/features/issues/353)` or something close to that.

Reply:
whoops yeah I missed that completely!